modules/virt-hot-plugging-bridge-network-interface-cli.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-hot-plugging-network-interfaces.adoc              
+
+:_content-type: PROCEDURE                                             
+[id="virt-hot-plugging-bridge-network-interface_{context}"]                                   
+= Hot plugging a bridge network interface using the CLI
+
+Hot plug a bridge network interface to a virtual machine (VM) while the VM is running. 
+
+.Prerequisites
+
+* The Multus Container Network Interface (CNI) plugin is deployed on the cluster.
+* A network attachment definition is configured in the same namespace as your VM.
+* You have installed the `virtctl` tool.
+
+.Procedure
+
+. If the VM to which you want to hot plug the network interface is not running, start it by using the following command:
++
+[source,terminal]
+----
+$ virtctl start <vm_name>
+----
++
+.Example VM configuration
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: vm-fedora
+spec:
+  running: true
+  template:
+    spec:
+      domain:
+        devices:
+          interfaces:
+          - masquerade: {}
+            name: defaultnetwork
+      # ...
+      networks:
+      - name: defaultnetwork
+        pod: {}
+# ...
+----
+
+. Use the following command to hot plug a new network interface to the running VM. The `virtctl addinterface` command adds the new network interface to the VM and virtual machine instance (VMI) specification but does not attach it to the running VM. 
++
+[source,terminal]
+----
+$ virtctl addinterface <vm_name> --network-attachment-definition-name <net_attach_def_name> --name <interface_name>
+----
++
+where:
+
+<vm_name>:: Specifies the name of the `VirtualMachine` object.
+<net_attach_def_name>::  Specifies the name of `NetworkAttachmentDefinition` object.
+<interface_name>:: Specifies the name of the new network interface.
+
+
+. To attach the network interface to the running VM, live migrate the VM by using the following command:
++
+[source,terminal]
+----
+$ virtctl migrate <vm_name>
+----
+
+.Verification
+
+. Verify that the VM live migration is successful by using the following command:
++
+[source,terminal]
+----
+$ oc get VirtualMachineInstanceMigration -w
+----
++
+.Example output
+[source,terminal]
+----
+NAME                        PHASE             VMI
+kubevirt-migrate-vm-lj62q   Scheduling        vm-fedora
+kubevirt-migrate-vm-lj62q   Scheduled         vm-fedora
+kubevirt-migrate-vm-lj62q   PreparingTarget   vm-fedora
+kubevirt-migrate-vm-lj62q   TargetReady       vm-fedora
+kubevirt-migrate-vm-lj62q   Running           vm-fedora
+kubevirt-migrate-vm-lj62q   Succeeded         vm-fedora
+----
+
+. Verify that the new interface is added to the VM by checking the VMI status:
++
+[source,terminal]
+----
+$ oc get vmi vm-fedora -ojsonpath="{ @.status.interfaces }"
```

Comment:
Is `-ojsonpath`  a typo? Looks like it should be `-o jsonpath`.

Reply:
`-ojsonpath` also applies.
Just like `-oyaml`, `-owide`, `oc get pod -n<namespace-name>`, etc.